src/redesign/register/RegistrationPage.jsx

```diff
@@ -400,7 +400,7 @@ class RegistrationPage extends React.Component {
               onClick={e => { this.handleSuggestionClick(e, this.state.emailErrorSuggestion); }}
             >
               {this.state.emailErrorSuggestion}
-            </Alert.Link>?
+            </Alert.Link>?<Icon src={Close} className="alert-close" onClick={this.handleOnClose} tabIndex="0" />
```

Comment:
I guess it should be an `IconButton` otherwise, nothing will happen on hover.

Reply:
Using `IconButton` background color will display on hover. We are using `alert-close` custom class in which `cursor: pointer`.